experiments/targeting_test.go

```diff
@@ -612,6 +612,156 @@ func TestComparisonNodeBadInput(t *testing.T) {
 	}
 }
 
+func TestNumberTypes(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name      string
+		targeting []byte
+		input     any
+	}{
+		{
+			name:      "gt-node-int",
```

Comment:
🔕 same thing about ordering here. Also it seems like you could probably simplify the test to have just an input of the following, and then just construct the two `[]byte` below since they're always the same.
```
{
  name: "node-***",
  input: ***(5),
}
```

Reply:
I actually prefer having both the input and the output contained within each test case.